website/content/docs/install.mdx

```diff
@@ -165,6 +165,25 @@ When running the Docker image, include the flag `--memory-swappiness=0`.
 
 [The swap space on macOS is always encrypted](https://support.apple.com/guide/mac-help/what-is-secure-virtual-memory-on-mac-mh11852/mac).
 
+## Checksum Verification
+
+### SHA-256
+
+Download the release and checksum files, and then verify the checksum using `sha256sum`:
+
+```
+$ OS="freebsd"
+$ ARCH="x86_64"
+$ PLATFORM="$OS_$ARCH"
+$ VERSION=$(curl -s https://api.github.com/repos/openbao/openbao/releases | jq -r '.[0] .tag_name' | cut -d'v' -f2-)
+$ RELEASE="bao_$VERSION_$PLATFORM.tar.gz"
+$ wget https://github.com/openbao/openbao/releases/download/v$VERSION/bao_$VERSION_$PLATFORM.tar.gz
+$ wget https://github.com/openbao/openbao/releases/download/v$VERSION/checksums-$OS.txt
+$ wget https://github.com/openbao/openbao/releases/download/v$VERSION/checksums-$OS.txt.gpgsig
+$ wget https://openbao.org/assets/openbao-gpg-pub-20240618.asc
```

Comment:
If we're downloading this file, should we use it to verify the `gpgsig` also downloaded above? Or should we just state the checksums' signature can be verified using a method below, and leave this and the above command off of this invocation?

Reply:
That's a good point - I've updated the docs to move `wget https://openbao.org/assets/openbao-gpg-pub-20240618.asc` to the Signature Verification section.